react/features/breakout-rooms/actions.js

```diff
@@ -217,6 +226,10 @@ export function moveToRoom(roomId?: string) {
             dispatch(setAudioMuted(audio.muted));
             dispatch(setVideoMuted(video.muted));
         } else {
+            const localTracks = getLocalTracks(getState()['features/base/tracks']);
```

Comment:
Is it possible there are no tracks at this point? As we are destroying them .. ooo we are destroying them when conference changes .... but we are still not there yet

Reply:
Exactly. It's possible they are empty if the user is muted from the start I guess, but that's covered too.